docs/docs/guides/basics/sign_and_send_tx/wallet_of_eth_node.md

```diff
@@ -169,6 +169,7 @@ import { Web3 } from 'web3';
 const web3 = new Web3(/* PROVIDER*/);
 
 // Second step: add an account to the Ethereum node and unlock it
+// IMPORTANT: never unlock an account on the mainnet. Use this only with your local dev node.
```

Comment:
There are very slim or no chances of any one using wallet inside Node usually, It was used in past for dev networks on local mostly, Now its deprecated https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-personal

Reply:
I just saw this dangerous code and added a warning. Yes, it would better to remove the usage of `personal`.
I create an issue: https://github.com/web3/web3.js/issues/6607. And I will delete this warning from this MR as it is out of it scope.